events/2019/05-contributor-summit/sig-f2f-guidelines.md

```diff
@@ -0,0 +1,34 @@
+# SIG F2F Guidelines for Contributor Summit Barcelona 2019
+
+SIG Chairs should register interest in a F2F during the Contributor Summit in Barcelona on Monday, May 20th 2019. **Important: We will try to accommodate all requests, but space is limited. UPDATE: We are at room capacity and cannot accept new requests.** Details about status of accepted F2F meetings can be found in [Issue #3358](https://github.com/kubernetes/community/issues/3358)
```

Comment:
```suggestion
SIG Chairs should register interest in a F2F for the Contributor Summit in Barcelona by Monday, May 20th 2019. **Important: We will try to accommodate all requests, but space is limited. UPDATE: We are at room capacity and cannot accept new requests.** Details about status of accepted F2F meetings can be found in [Issue #3358](https://github.com/kubernetes/community/issues/3358)
```
nit on the sentence :x

Reply:
Thanks! I added a commit to make this change :)